src/client/debug/main/Main.svelte

```diff
@@ -36,6 +38,11 @@
     color: #888;
```

Comment:
Let’s remove the `.json` style block (and classes) as it’s no longer needed with the JSON tree styling.